temporalcli/commandsmd/commands.md

```diff
@@ -721,10 +721,35 @@ Queue command, run `temporal task-queue [command] [command options]`.
 
 Includes options set for [client](#options-set-for-client).
 
-### temporal task-queue describe: Provides task reachability and pollers information for Workers on this Task Queue.
+### temporal task-queue describe: Provides pollers information, backlog statistics, and task reachability (experimental) for Workers on this Task Queue.
 
-The `temporal task-queue describe` command provides task reachability information for the requested versions and all task types,
-which can be used to safely retire Workers with old code versions, provided that they were assigned a Build ID.
+The `temporal task-queue describe` command provides poller information, backlog statistics and task reachability (experimental) 
+information for the requested versions and all task types, which can be used to safely retire Workers with old code versions, 
+provided that they were assigned a Build ID.
```

Comment:
this part of the sentence is only about reachability and it should be removed.

Reply:
I agree, however we should have the part : `the requested versions and all task types`. 

All in all, the sentence should look like:
The `temporal task-queue describe` command provides poller information, backlog statistics and task reachability (experimental) information for the requested versions and task types.

Reply:
> task reachability (experimental) ~information~

"information" seems redundant?